sdk/boms/spring-cloud-azure-dependencies/pom.xml

```diff
@@ -76,207 +76,207 @@
       <dependency>
         <groupId>com.azure.spring</groupId>
         <artifactId>spring-cloud-azure-actuator</artifactId>
-        <version>4.2.0-beta.1</version> <!-- {x-version-update;com.azure.spring:spring-cloud-azure-actuator;current} -->
+        <version>4.1.0</version> <!-- {x-version-update;com.azure.spring:spring-cloud-azure-actuator;dependency} -->
```

Comment:
FYI: This will be updated by the auto created PR, before release `spring-cloud-azure-dependencies`. I'll write this into the release wiki.

Reply:
I think these dependencies' tag status could be aligned with that of `spring-cloud-azure-dependencies`, then all of them could be udpated at the same pr.

Reply:
It's restricted by the tag: `<!-- {x-version-update;com.azure.spring:spring-cloud-azure-actuator;dependency} -->`.

1. If we use `dependency`, it should be `4.1.0`. `4.2.0` will cause pipeline failure.
2. If we use `current`, it's error-prone, because the auto-created PR may update the version to next beta version automatically.

So, I choose `dependency`, merge the auto-created PR before release `spring-cloud-azure-dependencies`.

Reply:
If we set allof them as current, then we can release `spring-cloud-azure-dependencies` from the commit of this pr, would it also be error-prone for you?
Besides, if we set different statuses for the bom artifact and dependencies, then wouldn't it also make the bom in an error state? I mean a 4.2.0 bom with 4.1.0 dependencies.

Reply:
Updated.